custom_model_runner/datarobot_drum/drum/model_adapter.py

```diff
@@ -122,12 +137,16 @@ def load_model_from_artifact(self):
         ):
             self._find_predictor_to_use()
 
-        # TODO: RAPTOR-4014 we eventually won't require this hook for some languages/frameworks
         if (
             self._target_type == TargetType.TRANSFORM
             and not self._custom_hooks[CustomHooks.TRANSFORM]
+            # don't require transform hook if sklearn transformer
+            and "sklearn" not in str(type(self._model))
```

Comment:
is this how estimators do it??

Reply:
estimators check whether the object is an instance of BaseEstimator, but there's a bunch of potential transformer objects for sklearn....I can check whether it has a `transform` method as well (or instead). or we can decide exactly which sklearn classes should be supported and check that it's one of those. or some combination thereof?